synapse/lib/cell.py

```diff
@@ -4439,6 +4446,13 @@ async def getCellInfo(self):
         Returns:
             Dict: A Dictionary of metadata.
         '''
+
+        mirror = self.conf.get('mirror')
```

Comment:
war on if/else ;)
```
mirror = self.conf.get('mirror', '')
if mirror:
    mirror = s_urlhelp.sanitizeUrl(mirror)
```

Reply:
Your suggestion changes the behavior of the returned value below. Seems like `None` might be preferrable since consumers of the cell info are likely to do something like `url = cellinfo.get('uplink:url'); if url is None: ...`.

Reply:
Just checking that we really want `uplink:url` in the cell info to be an empty string instead of None?

Reply:
I could go either way. Honestly a `None` is just as clear that there is NO mirror vs have an empty string.